src/crawlee/basic_crawler/basic_crawler.py

```diff
@@ -379,7 +379,8 @@ def sigint_handler() -> None:
         self._has_finished_before = True
 
         final_statistics = self._statistics.calculate()
-        self._logger.info(f'Final request statistics: {final_statistics}')
+        self._logger.info('Final request statistics: ')
+        final_statistics.to_table()
```

Comment:
The table should be concatenated with the logged message. What the `self._logger.info` call does depends on the configuration of the `logging` system, which is open to library users. Therefore, the two parts of the log line may end up in different places.